benchmarks/temp/code/aa.py

```diff
@@ -0,0 +1,10 @@
+import time
+
+for i in range(100):
+  print 'Hello World'
+
+str = raw_input("What is your name?: ")
```

Comment:
As grondinjc pointed me out, str is a class, so maybe we should use another variable name, just to be safe.